apps/backend/src/modules/contributors/contributor.service.ts

```diff
@@ -164,6 +166,57 @@ class ContributorService implements Service {
 		return item.toObject();
 	}
 
+	public async split(
+		contributorId: number,
+		payload: ContributorSplitRequestDto,
+	): Promise<ContributorGetAllItemResponseDto | null> {
+		const currentContributorEntity =
+			await this.contributorRepository.find(contributorId);
+		const hasContributor = currentContributorEntity !== null;
+
+		if (!hasContributor) {
+			throw new ContributorError({
+				message: ExceptionMessage.CONTRIBUTOR_NOT_FOUND,
+				status: HTTPCode.NOT_FOUND,
+			});
+		}
+
+		const currentContributor = currentContributorEntity.toObject();
+
+		if (currentContributor.gitEmails.length <= SINGLE_ITEM) {
+			throw new ContributorError({
+				message: ExceptionMessage.CONTRIBUTOR_SPLIT_SINGLE_EMAIL,
+				status: HTTPCode.CONFLICT,
+			});
+		}
+
+		const splittedEmail = currentContributor.gitEmails.find(
+			({ id }) => id === payload.emailId,
+		);
+		const hasSplittedEmail = splittedEmail !== undefined;
```

Comment:
should be split in 3d form
```suggestion
		const splitEmail = currentContributor.gitEmails.find(
			({ id }) => id === payload.emailId,
		);
		const splitEmail = splitEmail !== undefined;
```